client/blocks/login/login-form.jsx

```diff
@@ -84,6 +84,7 @@ export class LoginForm extends Component {
 			if ( window && window.calypsoLoadStartTime && window.calypsoLoadStartTime[ 'log-in' ] ) {
 				this.props.recordTracksEvent( 'calypso_load_end', {
 					elapsed_ms: window.performance.now() - window.calypsoLoadStartTime[ 'log-in' ].startTimestamp,
+					load_id: window.calypsoLoadStartTime[ 'log-in' ].loadId,
```

Comment:
Shouldn't it be `/log-in` here, given that you prepend a `/` here https://github.com/Automattic/wp-calypso/pull/21963/files#diff-a49b840027f7d1056bdfe24d63d58f2bR296

Reply:
Nope, the key is `pathName`: https://github.com/Automattic/wp-calypso/pull/21963/files#diff-a49b840027f7d1056bdfe24d63d58f2bR293

Reply:
Right 👍